### PR TITLE
MBMS-39209 Fixed Step Header Capitalizaiton Issues

### PR DESCRIPTION
**Pull request by ConnorJeff**
# [MBMS-39209](https://vajira.max.gov/browse/MBMS-39029)
**Click link above to be directed to Jira**

## Summary
Details: Most of the step titles at the top of the pre-need application pages are in title case.

Issue: The step titles need to be corrected to be in sentence case per va.gov content guidance for titles.

Feature Flag Status : Does not impact

Steps to reproduce:

1. Navigate to pre-need application

2. Start application by clicking "Sign in to start your application" button or "Start your application without signing in" link.

3. Navigate to any step in the form, either through the Veteran/service member flow or the non-veteran/non-service member flows. The only correctly formatted step title is for Supporting Documents.

**Note:** There is an extra step added to the flows for non-Veterans/non-service members. Veterans/service members go through 6 total steps in their flow, and non-veterans/non-service members go through 7 steps before submitting application.

## Expected results
Step headers inside application need to be updated to sentence casing instead of being title case per va.gov content standards. See for further guidance: [https://design.va.gov/content-style-guide/capitalization ](https://design.va.gov/content-style-guide/capitalization )

**Beginning of application after clicking button/link to start application:**

- Step 1 of 7: Applicant Information: Change casing to Step 1 of 7: Applicant information

 
**Service Member/Veteran Flow (after selecting Relationship to Service Member):**

- Step 1 of 6: Applicant Information: Change casing to Step 1 of 6: Applicant information
- Step 2 of 6: Military History: Change casing to Step 2 of 6: Military history
- Step 3 of 6: Burial Benefits: Change casing to Step 3 of 6: Burial benefits
- Step 5 of 6: Contact Information: Change casing to Step 5 of 6: Contact information
- Step 6 of 6: Review Application: Change casing to Step 6 of 6: Review application

 
**Non-Service Member/Non-Veteran Flows (after selecting Relationship to Service Member):**

- Step 1 of 7: Applicant Information: Change casing to Step 1 of 7: Applicant information
- Step 2 of 7: Sponsor Information: Change casing to Step 2 of 7: Sponsor information
- Step 3 of 7: Military History: Change casing to Step 3 of 7: Military history
- Step 4 of 7: Burial Benefits: Change casing to Step 4 of 7: Burial benefits
- Step 6 of 7: Contact Information: Change casing to Step 6 of 7: Contact information
- Step 7 of 7: Review Application: Change casing to Step 7 of 7: Review application

## Testing done

- Local Testing

## What areas of the site does it impact?
Preneed Eligibility Form

## Acceptance criteria

- [x] Captitalization Issues Fixed for **Service Member/Veteran Flow**
- [x] Captitalization Issues Fixed for **Non-Service Member/Non-Veteran Flows**

### CODE REVIEW

**Review by DouglasGilbert1 — APPROVED** [review]

File: src/applications/pre-need/config/form.jsx [hunk 1]
```diff
@@ -657,7 +663,7 @@ const formConfig = {
       },
     },
     supportingDocuments: {
-      title: 'Supporting documents',
+      title: 'Supporting Documents',
```
Comment (ksantiagoBAH) [inline src/applications/pre-need/config/form.jsx:666]: Forgot one flag here

**Review by ksantiagoBAH — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

File: src/applications/pre-need/config/form.jsx
[the remark this answers is not in the document]
Comment (ConnorJeff) [reply src/applications/pre-need/config/form.jsx:1]: I misspelled Sponsor. And the purpose of the bug is to change the capitalization, and becuase it is a funcitonal change I believe it is supposed to be behind a production flag. I will go ahead and change the spelling of Sponsor.

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**